### ISSUE TITLE
Custom versioning strategy - is it supposed to set app bundle?

### ISSUE THREAD

**Issue by gpiffaretti** [webbertakken/unity-builder#146]
Hey! I'm using this action and it's awesome.

I'm trying an action for handling versions and changelogs, so I want to use custom versioning strategy in the Unity Builder.

Here's part of my setup:

```   
        # Generate changelog with conventional commits
        - name: Conventional Changelog Action
          id: changelog
          uses: TriPSs/conventional-changelog-action@v3
          with:
            github-token: ${{ secrets.github_token }}
            skip-commit: "true"
            output-file: "false"

        # Build APK
        - name: Build APK with Unity
          uses: webbertakken/unity-builder@v0.10
          with:
            versioning: Custom
            version: ${{ steps.changelog.outputs.tag }}
            projectPath: ${{ matrix.projectPath }}
            unityVersion: ${{ matrix.unityVersion }}
            targetPlatform: ${{ matrix.targetPlatform }}
            buildsPath: build
            buildName: androidbuild
```

My questions are:
- Am I using the correct inputs? Docs mention the version field but there's no example.
- What is that value used for? I created APKs and the app bundle is still 0.1, which is the default value in the Unity project settings. Should this value be overriden by Unity Builder versioning? I don't understand what this is used for. I'm uploading the builds to AppCenter and later will go to the stores and I'd like them to get the version number from Unity Builder.

**Comment by webbertakken** [comment]
Hi @gpiffaretti, and thank you for your report.

First off; you're using a pretty old version of unity-builder, lots has been improved since then and some of the things you read in the docs may not even be available in v0.10 yet.

> Am I using the correct inputs? 

Yes I think so, but if you use the custom strategy it's up to you to provide the correct version number.
What does your tag looks like?

> Docs mention the version field but there's no example.

Note that version is by default configured, so you don't have to do anything with this input unless you want to override the default behaviour.

Could you be very specific about what you would like to see for the example? I think it may be worth adding this to the docs.

> What is that value used for? I created APKs and the app bundle is still 0.1, which is the default value in the Unity project settings. Should this value be overriden by Unity Builder versioning?

Yes, but I suspect this didn't happen because you're inputting some string that is not compatible with the version field. Maybe you could try omitting the `v` part from `v0.0.0`.

> I'm uploading the builds to AppCenter and later will go to the stores and I'd like them to get the version number from Unity Builder.

Have you considered using builders default versioning? Might be worth considering as people have adapted it  to update the correct versions throughout.

**Comment by webbertakken** [comment]
Hope that answers your question. Feel free to reopen if not.
